lib/Doctrine/ORM/Query/Expr.php

```diff
@@ -328,6 +328,19 @@ public function abs($x)
         return new Expr\Func('ABS', [$x]);
     }
 
+    /**
+     * Creates a MOD($x, $y) function expression to return the remainder of $x divided by $y.
+     *
+     * @param mixed $x
+     * @param mixed $y
+     *
+     * @return Expr\Func
```

Comment:
I would remove this and use a return type declaration here, regardless of the context, not sure what other contributors think. For me it's just more code to migrate.